trl/trainer/grpo_trainer.py

```diff
@@ -185,6 +185,48 @@ def __len__(self) -> int:
         return (self.num_samples // self.batch_size) * self.batch_size * self.mini_repeat_count * self.repeat_count
 
 
+class EntropyController:
```

Comment:
I don't think this controller is needed. Maybe it's simpler with only attributes in GRPO?